dotsloader/build.gradle

```diff
@@ -46,8 +40,9 @@ android {
 }
 
 dependencies {
-    javadocDeps 'com.android.support:support-annotations:25.1.0'
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    javadocDeps 'com.android.support:support-annotations:28.0.0'
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:28.0.0'
```

Comment:
try to avoid any dependency in the library. We can work even without the support library.
The only benefit we get out of this support library is the library is "ContextCompat.getColor" which we can use directly also.